.github/workflows/playwright.yml

```diff
@@ -0,0 +1,78 @@
+name: Playwright Tests
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+  schedule:
+    - cron: "0 2 * * *"
+
```

Comment:
Should we add a dispatch for `pull_request` as well?

Reply:
Now I install latest available tag - so it does not make sense on PRs if they are not tagged.
I would like to have some quick workflow for PRs and more complex for tags, but it's just an idea for now.